recipes/jdcal/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "jdcal" %}
+{% set version = "1.3" %}
+{% set sha256 = "b760160f8dc8cc51d17875c6b663fafe64be699e10ce34b6a95184b5aa0fdc9e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - jdcal
+    - jdcal.fpart
+    - jdcal.ipart
+    - jdcal.is_leap
+    - jdcal.gcal2jd
+    - jdcal.jd2gcal
+    - jdcal.jcal2jd
+    - jdcal.jd2jcal
+    - jdcal.MJD_0
+
+about:
+  home: https://github.com/phn/jdcal 
+  license: BSD
```

Comment:
Please specify license version (e.g. `BSD 2-Clause` for the 2-Clause version of the BSD license).